src/utils/validation.ts

```diff
@@ -36,6 +36,17 @@ export const validateIsFutureDate = (value: string) => {
   return isFuture(parsedDate) ? undefined : 'Card expired';
 };
 
+export const isPositiveInteger = (entity: string) => (value: string) => {
+  const positiveInteger = /^(?:[0-9]+\/[0-9]+|[0-9]+|[a-zA-Z]+|\/)+$/;
```

Comment:
Also make me simpler: use `isInteger` from `lodash` and check that's bigger than 0